manifests/1/1ic/BPMN-RPAstudio/21.8.2/1ic.BPMN-RPAstudio.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: 1ic.BPMN-RPAstudio
+PackageVersion: 21.8.2
+MinimumOSVersion: 10.0.0.0
+FileExtensions:
+- .flw
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+#  Scope: 
+  InstallerUrl: http://1ic.nl/BPMN_RPA/BPMN_RPA_studio.exe
+  InstallerSha256: 603FC56E4CF6602668D6F9C5264A8DB11F2C637F18E6FDA54CA47D2201EFD340
+  InstallerSwitches:
+    Custom: /norestart
+    Silent: /S
+    SilentWithProgress: /S
```

It doesn't look like it needs the `/norestart` switch. Also, it seems like the silent install is `/silent`

```suggestion
    Silent: /Silent
    SilentWithProgress: /Silent
```

---

OK. Do I need to change it and make a new pull request?

---

You should be able to commit my suggestion.